app/validators/uuid_validator.rb

```diff
@@ -0,0 +1,10 @@
+# See README.md for copyright details
+
+class UuidValidator < ActiveModel::EachValidator
+
+  def validate_each(record, attribute, value)
+    unless /^[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}$/i.match(value)
```

Comment:
Could the validate method of uuid be useful here? http://www.rubydoc.info/gems/uuid/2.3.1/UUID.validate
The problem is that it allows different uuid formats, but if that is not a problem for the project, it could be better to delegate to the uuid gem this regular expression